utilities/update_versions.sh

```diff
@@ -1,278 +0,0 @@
-#!/bin/bash
```

Comment:
My masterpiece is dying... It was ahead of its time, too masterpiecy for its contemporaries.... =(...

Reply:
Naw, it's just that you chose bash as the language of your masterpiece :-P